worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/BukkitWorldConfiguration.java

```diff
@@ -73,6 +73,8 @@
     public boolean usePaperEntityOrigin;
     /* Configuration data end */
 
+    public boolean needsBlockListener;
```

Comment:
name is rather inaccurate since it's just for physics, also should be @unreported

Reply:
https://github.com/EngineHub/WorldGuard/pull/1780#issuecomment-860891825